docs/rest-apis/gateway/llm-provider-management.md

```
@@ -92,18 +92,56 @@ Required roles: `admin`
+        },
+        {
+          "path": "/models/{modelId}",
+          "methods": [
+            "GET"
+          ]
+        }
+      ]
+    }
+  }
 }
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)